forms.py

```diff
@@ -445,8 +439,8 @@ def save(self, event, handler=None):
             maxwinners=self.cleaned_data['maxwinners'],
             extrainfo=self.cleaned_data['extrainfo'],
             estimatedvalue=self.cleaned_data['estimatedvalue'],
-            minimumbid=self.cleaned_data['suggestedamount'],
-            maximumbid=self.cleaned_data['suggestedamount'],
```

Comment:
GDQ has a minimum donation of $5, which might explain these hardcoded values. If we have a different minimum, we'll need to change this number (and probably change it in other places as well?).